### ISSUE TITLE
Footnote rule length

### ISSUE THREAD

**Issue by borisveytsman** [borisveytsman/acmart#45]
Frank Mittelbach objects agains full column footnote rule.  Consider options.

long line on first column and accept the difference

have the real footnotes above the copyright

move the real footnotes into the second column

**Comment by krono** [comment]
Re footnoterule: Tschichold's recommendation is around 4 cicero (~pica)

**Comment by MatthewFluet** [comment]
I don't understand the different options listed.  Is the first option suggesting that the first page first column would use a different footnote rule (above copyright) than other pages/columns?  Is the second option the current behavior?  If the third option suggests moving title, author, and any first-column footnotes to the second column, then that doesn't seem right.

**Comment by borisveytsman** [comment]
In the first column we use column-wide footnote rules because they divide the copyright info from the rest.  To make the outlook uniform, I've made the other footnote rules wide.  Is this pleasant looking?  What can we do?

**Comment by borisveytsman** [comment]
After long experiments I've adopted the behavior of the old ACM classes.

1.  Copyright/permission block is separated by a full width rule from the rest.
2.  "Normal" footnotes as well as title and author notes are separated by a shorter rule
3.  If there are normal footnotes in the first column, we use both separator.

I've added the following:  \thanks, as different from title and author notes, belong to the copyright/permission block, since they refer to the paper as a whole rather than to its elements like title and author.
